bin/prepare_rt_kernel.sh

```diff
@@ -5,8 +5,9 @@ RT_PATCH_VERSION="4.13.13-rt5"
 KERNEL_DIR="v4.x"
 KERNEL_VERSION="4.13.13"
```

Comment:
Should these be "X.X.X" as for other script? Or make a seperate file that is sourced into both?